UnityProject/Assets/Scripts/Construction/SpawnOnWrench.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpawnOnWrench : MonoBehaviour, ICheckedInteractable<HandApply>
+{
+	public GameObject toSpawn = null;
```

Comment:
if this is meant to be set in inspector, use a private property with ``[SerializeField]`` decorator and a public getter. Also add a ``Tooltip`` decorator. 

[Guidelines reference](https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide/#component-and-scriptable-object-design)